### PR TITLE
Fix threshold save bug

### PR DESCRIPTION
**Pull request by ashleyclx**
Fixed bug where threshold was not properly updated when commands were given.
Also fixed some checkstyle.

### CODE REVIEW

File: src/main/java/seedu/address/model/library/LibraryLogic.java [hunk 1]
```diff
@@ -139,7 +140,7 @@ public boolean hasNoAvailableBooks() {
     public void saveBooksToFile(ReadOnlyLibrary library) throws IOException {
         createFileIfNotExists();
         try (PrintWriter writer = new PrintWriter(new FileWriter(filePath))) {
-            writer.println(threshold.toString());
+             writer.println(library.getThreshold());
```
Comment (Ty-stan0417) [inline src/main/java/seedu/address/model/library/LibraryLogic.java:143]: No whitespace, remember to follow the checkstyle

File: src/main/java/seedu/address/model/library/Library.java [hunk 2]
```diff
@@ -138,6 +138,11 @@ public boolean hasBookInLibrary(Book book) {
         return bookList.contains(book);
     }
 
+    /**
+     * Removes {@code book} from library book list.
+     * @param book book to be removed from library
+     * @return returns the book removed from library
+     */
```
Comment (Ty-stan0417) [inline src/main/java/seedu/address/model/library/Library.java:145]: Nice javadoc

**Review by Ty-stan0417 — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/model/library/Threshold.java [hunk 3]
```diff
@@ -1,6 +1,5 @@
 package seedu.address.model.library;
 
-import seedu.address.commons.core.index.Index;
 import seedu.address.model.person.MeritScore;
```
Comment (Darkarche3) [inline src/main/java/seedu/address/model/library/Threshold.java:3]: Well done removing unused imports

File: src/main/java/seedu/address/model/library/Library.java [hunk 4]
```diff
@@ -138,6 +138,11 @@ public boolean hasBookInLibrary(Book book) {
...
     public Book popBookFromLibrary(Book book) {
```
Comment (jinhanfromNUS) [inline src/main/java/seedu/address/model/library/Library.java:146]: I love the name of this method, it is very clear and easy to understand.

File: src/main/java/seedu/address/model/library/LibraryLogic.java (hunk 1 above)
Comment (jinhanfromNUS) [inline src/main/java/seedu/address/model/library/LibraryLogic.java:143]: Please remove the whitespace at the beginning :). Everything looks awesome tho

File: src/main/java/seedu/address/model/library/Threshold.java [hunk 5]
```diff
@@ -1,6 +1,5 @@
 package seedu.address.model.library;
 
-import seedu.address.commons.core.index.Index;
```
Comment (jinhanfromNUS) [inline src/main/java/seedu/address/model/library/Threshold.java:3]: Thanks for removing this!

File: src/main/java/seedu/address/model/library/Library.java [hunk 6]
```diff
@@ -138,6 +138,11 @@ public boolean hasBookInLibrary(Book book) {
         return bookList.contains(book);
     }
 
+    /**
```
Comment (jinhanfromNUS) [inline src/main/java/seedu/address/model/library/Library.java:141]: Thanks for writing this Java Docs. It is very clear.

**Pull request merged**